tools/schemas/changelog-urls-schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "patternProperties": {
+    "^[a-zA-Z]+$": {
```

Comment:
shouldn't we allow more, depNames can have `-/@` too, eg npm

Reply:
This regex is for the datasource. There's another below which is used for the depNames. I have updated it to include @.

Reply:
we need `-` for datasource 😁